### PR TITLE
Save files to Downloads, don't overwrite files

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

Also a little tweak to reserve space for the "show in finder"

### CODE REVIEW

File: shared/actions/chat.js [hunk 1]
```diff
@@ -551,7 +552,7 @@ function * _incomingMessage (action: IncomingMessage): SagaGenerator<any, any> {
           })
 
           if (message.type === 'Attachment' && !message.previewPath && message.messageID) {
-            yield put(loadAttachment(conversationIDKey, message.messageID, true, tmpFile(message.filename)))
+            yield put(loadAttachment(conversationIDKey, message.messageID, true, false, tmpFile('preview-' + message.filename)))
```
Comment (chrisnojima) [inline shared/actions/chat.js]: maybe we can use some metadata from the server to make a good name, like based on the hash or something. i'm worried about us redownloading stuff and not cleaning up the tmp folder and etc etc

**Comment by chrisnojima** [comment]
If i open the preview and click download and then click show in finder it shows me the temp file one and not the download path one

File: shared/actions/chat.js [hunk 2]
```diff
@@ -111,6 +111,10 @@ const _messageOutboxIDSelector = (state: TypedState, conversationIDKey: Conversa
 const _pendingFailureSelector = (state: TypedState, outboxID: OutboxIDKey) => state.chat.get('pendingFailures').get(outboxID)
 const _devicenameSelector = (state: TypedState) => state.config && state.config.extendedConfig && state.config.extendedConfig.device && state.config.extendedConfig.device.name
 
+function _tmpFileName (isHdPreview: boolean, messageID: ?MessageID, filename: string) {
+  return `kbchat-${isHdPreview ? 'hdPreview' : 'preview'}-${messageID || ''}-${filename}`
```
Comment (chrisnojima) [inline shared/actions/chat.js]: doesn't this need conversationid also? you could collide (rare but possible right)

**Comment by keybase-ci-visdiff** [comment]
The commits 0e12db8d0c42...a31b8d70aaf6 introduce visual changes on linux. <details><summary>:mag_right: 11 changed</summary>

 * Changed: **Attachment_Popup-You_Wide_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Popup-You_Wide_Image-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Popup-You_Wide_Image-a31b8d70aaf6.png)  
   Attachment_Popup-You_Wide_Image comparison
 * Changed: **Attachment_Popup-You_Small_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Popup-You_Small_Image-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Popup-You_Small_Image-a31b8d70aaf6.png)  
   Attachment_Popup-You_Small_Image comparison
 * Changed: **Attachment_Message-Basic_Uploading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Uploading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Uploading-a31b8d70aaf6.png)  
   Attachment_Message-Basic_Uploading comparison
 * Changed: **Attachment_Message-Basic_Preview_Image_Pending** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Preview_Image_Pending-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Preview_Image_Pending-a31b8d70aaf6.png)  
   Attachment_Message-Basic_Preview_Image_Pending comparison
 * Changed: **Attachment_Message-Basic_Preview_Image_Not_Downloaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Preview_Image_Not_Downloaded-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Preview_Image_Not_Downloaded-a31b8d70aaf6.png)  
   Attachment_Message-Basic_Preview_Image_Not_Downloaded comparison
 * Changed: **Attachment_Message-Basic_Preview_Image_Failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Preview_Image_Failed-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Preview_Image_Failed-a31b8d70aaf6.png)  
   Attachment_Message-Basic_Preview_Image_Failed comparison
 * Changed: **Attachment_Message-Basic_Not_loaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Not_loaded-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Not_loaded-a31b8d70aaf6.png)  
   Attachment_Message-Basic_Not_loaded comparison
 * Changed: **Attachment_Message-Basic_Generic_File_Uploading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Generic_File_Uploading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Generic_File_Uploading-a31b8d70aaf6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Generic_File_Uploading.png)
 * Changed: **Attachment_Message-Basic_Generic_File_Failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Generic_File_Failed-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Generic_File_Failed-a31b8d70aaf6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Generic_File_Failed.png)
 * Changed: **Attachment_Message-Basic_Generic_File_Downloading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Generic_File_Downloading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Generic_File_Downloading-a31b8d70aaf6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Generic_File_Downloading.png)
 * Changed: **Attachment_Message-Basic_Downloading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Downloading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Downloading-a31b8d70aaf6.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484949444057-0e12db8d0c42-a31b8d70aaf6-linux/Attachment_Message-Basic_Downloading.png)
</summary>

File: shared/actions/chat.js (hunk 2 above)
Comment (MarcoPolo) [inline shared/actions/chat.js]: 👍

**Comment by keybase-ci-visdiff** [comment]
The commits 0e12db8d0c42...a0cc7cb24a41 introduce visual changes on linux. <details><summary>:mag_right: 11 changed</summary>

 * Changed: **Attachment_Popup-You_Wide_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Popup-You_Wide_Image-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Popup-You_Wide_Image-a0cc7cb24a41.png)  
   Attachment_Popup-You_Wide_Image comparison
 * Changed: **Attachment_Popup-You_Small_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Popup-You_Small_Image-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Popup-You_Small_Image-a0cc7cb24a41.png)  
   Attachment_Popup-You_Small_Image comparison
 * Changed: **Attachment_Message-Basic_Uploading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Uploading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Uploading-a0cc7cb24a41.png)  
   Attachment_Message-Basic_Uploading comparison
 * Changed: **Attachment_Message-Basic_Preview_Image_Pending** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Preview_Image_Pending-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Preview_Image_Pending-a0cc7cb24a41.png)  
   Attachment_Message-Basic_Preview_Image_Pending comparison
 * Changed: **Attachment_Message-Basic_Preview_Image_Not_Downloaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Preview_Image_Not_Downloaded-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Preview_Image_Not_Downloaded-a0cc7cb24a41.png)  
   Attachment_Message-Basic_Preview_Image_Not_Downloaded comparison
 * Changed: **Attachment_Message-Basic_Preview_Image_Failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Preview_Image_Failed-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Preview_Image_Failed-a0cc7cb24a41.png)  
   Attachment_Message-Basic_Preview_Image_Failed comparison
 * Changed: **Attachment_Message-Basic_Not_loaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Not_loaded-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Not_loaded-a0cc7cb24a41.png)  
   Attachment_Message-Basic_Not_loaded comparison
 * Changed: **Attachment_Message-Basic_Generic_File_Uploading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Generic_File_Uploading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Generic_File_Uploading-a0cc7cb24a41.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Generic_File_Uploading.png)
 * Changed: **Attachment_Message-Basic_Generic_File_Failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Generic_File_Failed-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Generic_File_Failed-a0cc7cb24a41.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Generic_File_Failed.png)
 * Changed: **Attachment_Message-Basic_Generic_File_Downloading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Generic_File_Downloading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Generic_File_Downloading-a0cc7cb24a41.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Generic_File_Downloading.png)
 * Changed: **Attachment_Message-Basic_Downloading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Downloading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Downloading-a0cc7cb24a41.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950175275-0e12db8d0c42-a0cc7cb24a41-linux/Attachment_Message-Basic_Downloading.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 0e12db8d0c42...5bade817582c introduce visual changes on linux. <details><summary>:mag_right: 15 changed</summary>

 * Changed: **Popup_Attachment-Popup_Attachment_Downloaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Popup_Attachment-Popup_Attachment_Downloaded-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Popup_Attachment-Popup_Attachment_Downloaded-5bade817582c.png)  
   Popup_Attachment-Popup_Attachment_Downloaded comparison
 * Changed: **Menubar-Private_Async_Writing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Menubar-Private_Async_Writing-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Menubar-Private_Async_Writing-5bade817582c.png)  
   Menubar-Private_Async_Writing comparison
 * Changed: **Attachment_Popup-You_Wide_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Popup-You_Wide_Image-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Popup-You_Wide_Image-5bade817582c.png)  
   Attachment_Popup-You_Wide_Image comparison
 * Changed: **Attachment_Popup-You_Small_Image** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Popup-You_Small_Image-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Popup-You_Small_Image-5bade817582c.png)  
   Attachment_Popup-You_Small_Image comparison
 * Changed: **Attachment_Popup-You_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Popup-You_Popup_Showing-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Popup-You_Popup_Showing-5bade817582c.png)  
   Attachment_Popup-You_Popup_Showing comparison
 * Changed: **Attachment_Popup-Following_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Popup-Following_Popup_Showing-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Popup-Following_Popup_Showing-5bade817582c.png)  
   Attachment_Popup-Following_Popup_Showing comparison
 * Changed: **Attachment_Message-Basic_Uploading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Uploading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Uploading-5bade817582c.png)  
   Attachment_Message-Basic_Uploading comparison
 * Changed: **Attachment_Message-Basic_Preview_Image_Pending** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Preview_Image_Pending-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Preview_Image_Pending-5bade817582c.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Preview_Image_Pending.png)
 * Changed: **Attachment_Message-Basic_Preview_Image_Not_Downloaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Preview_Image_Not_Downloaded-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Preview_Image_Not_Downloaded-5bade817582c.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Preview_Image_Not_Downloaded.png)
 * Changed: **Attachment_Message-Basic_Preview_Image_Failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Preview_Image_Failed-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Preview_Image_Failed-5bade817582c.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Preview_Image_Failed.png)
 * Changed: **Attachment_Message-Basic_Not_loaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Not_loaded-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Not_loaded-5bade817582c.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Not_loaded.png)
 * Changed: **Attachment_Message-Basic_Generic_File_Uploading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Generic_File_Uploading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Generic_File_Uploading-5bade817582c.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Generic_File_Uploading.png)
 * Changed: **Attachment_Message-Basic_Generic_File_Failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Generic_File_Failed-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Generic_File_Failed-5bade817582c.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Generic_File_Failed.png)
 * Changed: **Attachment_Message-Basic_Generic_File_Downloading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Generic_File_Downloading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Generic_File_Downloading-5bade817582c.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Generic_File_Downloading.png)
 * Changed: **Attachment_Message-Basic_Downloading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Downloading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Downloading-5bade817582c.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1484950282092-0e12db8d0c42-5bade817582c-linux/Attachment_Message-Basic_Downloading.png)
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits 0e12db8d0c42...0da09894cd65 introduce visual changes on linux. <details><summary>:mag_right: 8 changed</summary>

 * Changed: **Popup_Attachment-Popup_Attachment_Downloaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Popup_Attachment-Popup_Attachment_Downloaded-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Popup_Attachment-Popup_Attachment_Downloaded-0da09894cd65.png)  
   Popup_Attachment-Popup_Attachment_Downloaded comparison
 * Changed: **Attachment_Popup-You_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Popup-You_Popup_Showing-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Popup-You_Popup_Showing-0da09894cd65.png)  
   Attachment_Popup-You_Popup_Showing comparison
 * Changed: **Attachment_Popup-Following_Popup_Showing** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Popup-Following_Popup_Showing-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Popup-Following_Popup_Showing-0da09894cd65.png)  
   Attachment_Popup-Following_Popup_Showing comparison
 * Changed: **Attachment_Message-Basic_Generic_File_sent** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_sent-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_sent-0da09894cd65.png)  
   Attachment_Message-Basic_Generic_File_sent comparison
 * Changed: **Attachment_Message-Basic_Generic_File_Uploading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_Uploading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_Uploading-0da09894cd65.png)  
   Attachment_Message-Basic_Generic_File_Uploading comparison
 * Changed: **Attachment_Message-Basic_Generic_File_Failed** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_Failed-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_Failed-0da09894cd65.png)  
   Attachment_Message-Basic_Generic_File_Failed comparison
 * Changed: **Attachment_Message-Basic_Generic_File_Downloading** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_Downloading-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_Downloading-0da09894cd65.png)  
   Attachment_Message-Basic_Generic_File_Downloading comparison
 * Changed: **Attachment_Message-Basic_Generic_File_Downloaded** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_Downloaded-0e12db8d0c42.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_Downloaded-0da09894cd65.png) [(diff)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1485206504754-0e12db8d0c42-0da09894cd65-linux/Attachment_Message-Basic_Generic_File_Downloaded.png)
</summary>

**Pull request merged**